reference/generators/cmake_find_package.rst

```diff
@@ -55,8 +55,56 @@ utilities exported by the package available for consumers just by setting `find_
++------------------------------------+-----------------------------------------------------------------------------------------------------+
+| <COMP-NAME>_COMPILE_DEFINITIONS    | Definitions of the component                                                                        |
++------------------------------------+-----------------------------------------------------------------------------------------------------+
+| <COMP-NAME>_COMPILE_OPTIONS_LIST   | Compile option flags of the component                                                               |
++------------------------------------+-----------------------------------------------------------------------------------------------------+
+| <COMP-NAME>_LINKER_FLAGS_LIST      | Definitions of the component                                                                        |
++------------------------------------+-----------------------------------------------------------------------------------------------------+
+| <COMP-NAME>_LIBS                   | Library paths to link                                                                               |
++------------------------------------+-----------------------------------------------------------------------------------------------------+
+| <COMP-NAME>_SYSTEM_LIBS            | System libraries to link                                                                            |
++------------------------------------+-----------------------------------------------------------------------------------------------------+
+| <COMP-NAME>_BUILD_MODULES_PATHS    | List of CMake module files with functionalities for consumers                                       |
++------------------------------------+-----------------------------------------------------------------------------------------------------+
+| <COMP-NAME>_FRAMEWORKS             | Framework names to do a `find_library()`                                                            |
++------------------------------------+-----------------------------------------------------------------------------------------------------+
+| <COMP-NAME>_FRAMEWORK_DIRS         | Framework directories to perform the `find_library()` of <PKG-NAME>_FRAMEWORKS                      |
++------------------------------------+-----------------------------------------------------------------------------------------------------+
+| <COMP-NAME>_DEPENDENCIES           | List of requirements                                                                                |
++------------------------------------+-----------------------------------------------------------------------------------------------------+
+
+Being ``<COMP-NAME>`` the component name used as key in the components dictionary (by default) or the one declared in ``cpp_info.name`` or
+in ``cpp_info.names["cmake_find_package"]`` if specified
```

Comment:
Here it should be  `self.cpp_info.components["cmp"].name` and `self.cpp_info.components["cmp"].names["cmake_find_package "]`. Probably too long but... 🤷